test/integration/laser.cc

```diff
@@ -608,6 +616,59 @@ TEST_P(LaserTest, LaserNoise)
   LaserUnitNoise(GetParam());
 }
 
+void LaserTest::LaserStrictUpdateRate(const std::string &_physicsEngine)
```

Comment:
There is one warning on CI about unused parameter `_physicsEngine`. I must have missed it in gazebo11.

Reply:
removed unused param in 95855f8